app/bundles/PluginBundle/Views/FormTheme/Integration/fields_row.html.php

```diff
@@ -83,26 +83,36 @@ class="<?php echo $child->vars['attr']['class']; ?>"
                         <div class="choice-wrapper">
                             <div class="btn-group btn-block" data-toggle="buttons">
                                 <?php $checked = $child->vars['value'] === '0'; ?>
-                                <label class="btn btn-default<?php if ($checked): echo ' active'; endif; ?>">
+                                <label class="btn-arrow<?php echo $indexCount; ?> btn btn-default<?php if ($checked): echo ' active'; endif; ?> <?php if ($child->vars['attr']['disabled']) {
+                    echo 'disabled';
+                } ?>">
```

Comment:
Can you change the if syntax to `<?php echo ($child->vars['attr']['disabled']) ? 'disabled' : '' ?>` so it emulates the format of a HTML tag?

Similarly the other 3 ifs in this file.